src/Pages/Sign/signUp.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import Nav from "../../Components/Nav";
+import InputWithLabel from "./InputWithLabel";
+import Terms from "./Terms";
+import Footer from "../../Components/Footer";
+import "./SignUp.scss";
+
+class SignUp extends Component {
+  constructor() {
+    super();
+    this.state = {
+      inputValue: {
+        id: "",
+        password: "",
+        confirm: "",
+        name: "",
+      },
+    };
+  }
+
+  getInputValue = (label, value) => {
+    const { inputValue } = this.state;
+
+    switch (label) {
+      case "아이디":
+        this.setState({ inputValue: { ...inputValue, id: value } });
+        break;
+      case "비밀번호":
+        this.setState({ inputValue: { ...inputValue, password: value } });
+        break;
+      case "비밀번호 확인":
+        this.setState({ inputValue: { ...inputValue, confirm: value } });
+        break;
+      case "이름":
+        this.setState({ inputValue: { ...inputValue, name: value } });
+        break;
+      default:
+        break;
+    }
+  };
```

Comment:
```suggestion
  getInputValue = (label, value) => {
    const { inputValue } = this.state;

    this.setState({ inputValue: { ...inputValue, [label]: value } })
  };
```
애초에 label 값을 한글이 아니라 키값을 그대로 넣어준다면 이렇게 한줄로 끝날거 같네요!